documentation/howto/maven_basics.adoc

```diff
@@ -0,0 +1,147 @@
+= Basic Maven Configuration
+:page-layout: howto
+:page-tab: docs
+:page-status: green
+:experimental:
+:imagesdir: ./images
+
+In the context of application development, Maven allows you to extend your applications by referencing and utilizing resources from local and remote repositories. This article serves as an introduction to using Maven with the IDE and contains instructions for the following tasks:
+
+. <<New_Project,Create a New Maven Project>>
+. <<New_Module,Create a New Maven Module>>
+. <<Maven_Support,Add Maven Support to an Existing Non-Maven Project>>
+
+Maven projects serve as parents for Maven modules. For each Maven module attached to a Maven project, a section is added to the project’s pom.xml file. Each command run by the Maven project is run against each of the included modules first.
```

Comment:
"parent", "attached" and "command" are not the correct terms in this case. And no, each command is not always run against each of the included modules first.

My suggestion: 
"Root Maven projects can serve as aggregators for multiple Maven modules. For each module that is part of a maven project, a `<module>` entry is added to the project's `pom.xml` file. A `pom.xml` that contains `<module>` entries are often referred to as an aggregator pom.

When modules are included into a project it is possible to execute Maven Goals across all of the modules by a single command."